app/actions/views/channel.js

```diff
@@ -690,4 +694,70 @@ function loadSidebar(data) {
             dispatch(batchActions(sidebarActions, 'BATCH_LOAD_SIDEBAR'));
         }
     };
+}
+
+export function loadUnreadChannelPosts() {
```

Comment:
maybe if you move this function to the `posts` actions file then you don't need so many `import from mattermost-redux/actions/posts`